### DISCUSSION TITLE
SOC alert remove

### DISCUSSION THREAD

**Discussion by b4b857f6ee** [Security-Onion-Solutions/securityonion#5899] [Unsupported Versions]
Hello, 

i try here and in the documentation and nothing is clear.
How we can remove all the alert. Clearing the logs didn't clear the alert.

Thank you :)

**Comment by dougburks** [comment]
Have you tried clicking the bell icon to acknowledge the alert?
https://docs.securityonion.net/en/2.3/alerts.html#data-table
